### PR TITLE
Add phone number support

### PR DESCRIPTION
**Pull request by piatrashkakanstantinass**
The regex is not 100% accurate; however, it works with many formats. It also provides the location for international numbers.

### CODE REVIEW

**Comment by piatrashkakanstantinass** [comment]
Just saw #51, so this can be used only as a temporary solution.

File: tests/test_regex_identifier.py [hunk 1]
```diff
@@ -209,6 +209,30 @@ def test_email4():
...
+    res = r.check(["+662025550156"])
+    assert "Phone Number" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_phone_numer4():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["+356 202 555 0156"])
+    assert "Phone Number" in res[0]["Regex Pattern"]["Name"]
+
+
```
Comment (bee-san) [inline tests/test_regex_identifier.py:235]: I love these tests!!!! What do you think about changing `numer` to `number`? 😄

File: pywhat/printer.py [hunk 2]
```diff
@@ -40,6 +42,25 @@ def pretty_print(self, text: dict):
...
+                    locations = []
+                    for code in codes:
+                        if number.startswith(code["dial_code"]):
+                            locations.append(code["name"])
+                    if len(locations):
+                        description = (
+                            "Location(s)"
+                            + ": "
+                            + ", ".join(locations)
+                        )
```
[the remark this answers is not in the document]
Comment (piatrashkakanstantinass) [reply pywhat/printer.py:63]: Sure!

File: tests/test_regex_identifier.py (hunk 1 above)
Comment (piatrashkakanstantinass) [reply tests/test_regex_identifier.py:235]: whoops

File: pywhat/printer.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (piatrashkakanstantinass) [reply pywhat/printer.py:63]: @bee-san Do you mean moving the code into regex_identifier.py and overriding description there?

**Comment by piatrashkakanstantinass** [comment]
@bee-san Done!🙂

**Review by bee-san — APPROVED** [review]

**Comment by bee-san** [comment]
Wooo! Sorry for taking so long to review, I took a break. LGTM and the tests pass :-) !

**Pull request merged**